### ISSUE TITLE
Curriculum Accessibility Issues

### ISSUE THREAD

**Issue by ShaunSHamilton** [freeCodeCamp/freeCodeCamp#40873]
On .dev the widget(I do not know what to call it) which shows the bool of whether or not a section is complete, comes up as a clickable element and is indexable, because it is a button:
![image](https://user-images.githubusercontent.com/51722130/106597112-dbd52c00-654d-11eb-84fb-f107a9b5d392.png)

**Comment by sky3760000** [comment]
ok

**Comment by moT01** [comment]
There's some more discussion about the "widget" [here](https://github.com/freeCodeCamp/freeCodeCamp/issues/385690) - It may end up getting removed. If you claim the cert, the widget will take you to the cert - so it is clickable in that instance. We could render a button if it's claimed or a div if not. Or just make it a div and then, I'm pretty sure there's something to make that indexed.

**Comment by Sembauke** [comment]
@raisedadead I have a fix for the entire issue now

**Comment by ShaunSHamilton** [comment]
@Sembauke You have not yet opened a PR referencing this issue, but looking at your current linked commit, it is not quite what we want. The reply above:
> If you claim the cert, the widget will take you to the cert - so it is clickable in that instance. We could render a button if it's claimed or a div if not. Or just make it a div

That means, the element should be indexable, just not when it is not acting as a button. However, we have not yet decided on whether to leave the element as a button and alter its tab properties, or to change the element to something like a `div` when it is not clickable.

**Comment by Sembauke** [comment]
I'll leave it for discussion for now then! :) 

> @Sembauke You have not yet opened a PR referencing this issue, but looking at your current linked commit, it is not quite what we want. The reply above:
> 
> > If you claim the cert, the widget will take you to the cert - so it is clickable in that instance. We could render a button if it's claimed or a div if not. Or just make it a div
> 
> That means, the element should be indexable, just not when it is not acting as a button. However, we have not yet decided on whether to leave the element as a button and alter its tab properties, or to change the element to something like a `div` when it is not clickable.

I'll leave it for discussion for now then! :)
